### ISSUE TITLE
Blindspin Dependency breaks installation in CI/CD

### ISSUE THREAD

**Issue by ShahBinoy** [testcontainers/testcontainers-python#124]
Attempting to install the test container breaks due to blindspin.

All our build started failing now

```
[2020-12-11T07:08:36.725] + pip3 install -r common/test-containers.txt

[2020-12-11T07:08:37.292] Collecting testcontainers==3.1.0

[2020-12-11T07:08:37.292]   Downloading testcontainers-3.1.0.tar.gz (15 kB)

[2020-12-11T07:08:37.866] Requirement already satisfied: wrapt in /usr/local/lib/python3.6/dist-packages (from testcontainers==3.1.0->-r common/test-containers.txt (line 1)) (1.12.1)

[2020-12-11T07:08:37.866] Collecting blindspin

[2020-12-11T07:08:37.866]   Downloading blindspin-2.0.1.tar.gz (2.2 kB)

[2020-12-11T07:08:38.130]     ERROR: Command errored out with exit status 1:

[2020-12-11T07:08:38.130]      command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-4qbykvrm/blindspin_3af84003b4474b10bf0bafb7094e3fa8/setup.py'"'"'; __file__='"'"'/tmp/pip-install-4qbykvrm/blindspin_3af84003b4474b10bf0bafb7094e3fa8/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-4b6cz5at

[2020-12-11T07:08:38.130]          cwd: /tmp/pip-install-4qbykvrm/blindspin_3af84003b4474b10bf0bafb7094e3fa8/

[2020-12-11T07:08:38.130]     Complete output (7 lines):

[2020-12-11T07:08:38.130]     Traceback (most recent call last):

[2020-12-11T07:08:38.130]       File "<string>", line 1, in <module>

[2020-12-11T07:08:38.130]       File "/tmp/pip-install-4qbykvrm/blindspin_3af84003b4474b10bf0bafb7094e3fa8/setup.py", line 7, in <module>

[2020-12-11T07:08:38.130]         readme = f.read()

[2020-12-11T07:08:38.130]       File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode

[2020-12-11T07:08:38.130]         return codecs.ascii_decode(input, self.errors)[0]

[2020-12-11T07:08:38.130]     UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 189: ordinal not in range(128)

```

**Comment by tillahoffmann** [comment]
Should be fixed in 3.2.0 (out later today, see #125).

**Comment by ShahBinoy** [comment]
Thank You !!
